lib/grpnice.gi

```diff
@@ -836,6 +836,13 @@ end);
 AttributeMethodByNiceMonomorphism( Size,
     [ IsGroup ] );
 
+#############################################################################
+##
+#M  StructureDescription( <G> )
+##
+AttributeMethodByNiceMonomorphism( StructureDescription,
```

Comment:
I am sorry, but I have no idea what this is supposed to do. Does it mean the GAP will check if ````IsHandledByNiceMonomorphism```` is true, and if yes, then checks ````StructureDescription```` for the nice morphism image?